config/locales/en.yml

```diff
@@ -1228,6 +1236,7 @@ en:
   notice_unable_delete_time_entry: "Unable to delete time log entry."
   notice_unable_delete_version: "Unable to delete version."
   notice_user_missing_authentication_method: User has yet to chose a password or another way the sign in.
+  notice_value_information: "Your %{key_name} is: %{value}"
```

Comment:
Our comments were outdated:
You said you intentionally made this string more "reusable".

I personally don't think that translations are very good at providing reusability, because we know too few about the languages we are translating into. We already had problems for french translations, where the translator did not know the gender of an embedded property (e.g. `key_name`). If we stay specific here (e.g. only use the string for access tokens), such things become easier for translators, because the gender is easier to determine.

I guess this is only one example how translations can become weird through reuse.